include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -2013,14 +2016,15 @@ __pattern_set_symmetric_difference(__hetero_tag<_BackendTag> __tag, _ExecutionPo
     }
     else
     {
-        typedef typename std::iterator_traits<_OutputIterator>::value_type _ValueType;
+        typedef typename std::iterator_traits<_ForwardIterator1>::value_type _ValueType1;
+        typedef typename std::iterator_traits<_ForwardIterator2>::value_type _ValueType2;
```

Comment:
optional: While we are here, can we switch this to a `using`?

Reply:
I think it's a big deal - we may fix a lot of places like that.

Reply:
As I mentioned, fully optional, fine to leave this as is here.  We can discuss those changes in that PR.